prebid-mobile/adops-setup-rewarded-video-mopub.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page_v2
+title: Setup Rewarded Video for MoPub
+description: Setup Rewarded Video for MoPub
+pid: 1
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile-adops
+sidebarType: 3
+---
+
+# Step-by-Step Line Item Setup for Rewarded Video on MoPub
+
+* TOC
+{:toc }
+
+This page describes step-by-step instructions to set up rewarded-video line items on MoPub to be used with the Prebid Mobile SDK.  
+
+## Step 1. Add a line item  
+
+  In the *Add a line item* section:
+  1.  For the *Type and Priority* settings, select *Non-Guaranteed* as the type and set the priority to twelve. This ensures the line item will compete with all other demand.  
+  2. Set the Rate to the price you want to target.&nbsp;  
+  <img src="/assets/images/ad-ops/mopub/rewarded/mopub-lineitem-rewarded.png" width="1000" height="544">&nbsp;&nbsp;  
+  3. In the *Advanced Targeting* section, set the target for *Keywords* to `hb_pb:0.50`&nbsp;
+  <img src="/assets/images/ad-ops/mopub/rewarded/mopub-lineitem-advanced.png" width="1000" height="254">&nbsp;&nbsp;
+
+For each level of pricing granularity required, one line item/creative pairing will need to be set up.
+
+Line items must be set up to target custom keywords that include bid price information. The bid price keywords will contain how much the buyer bid on the impression.
+
+Prebid Mobile, by default, will send the highest bid price to MoPub using the keyword `hb_pb` but will also submit the keys `hb_pb_BIDDERCODE`. You can decide to create one set of line items for all bidders or one set of line items for each bidder.
```

Comment:
"keys hb_pb_BIDDERCODE" - should be "key", singular?

Reply:
I changed it to key(s) as there can be multiple bidders

Reply:
Actually, revised it to `but will also submit the key `hb_pb_BIDDERCODE` for each bidder`